hooks/settings/_utils.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from os import environ
+
+
+def get_env(variable, *, default=None, convert=None):
```

Comment:
@jimrkh **convert** isn't really needed, we don't encode anything in bytes here. :-)

Reply:
@harpener  I used standard convert function that we use in other projects but OK, I will remove it because it is not necessary here